app/account/page.tsx

```diff
@@ -0,0 +1,143 @@
+'use client'
+
+import {
+  Heading,
+  Flex,
+  Box,
+  Container,
+  VStack,
+  Image,
+  Text,
+  useColorModeValue,
+  Spacer
+} from '@chakra-ui/react'
+import { CiEdit } from 'react-icons/ci'
+import { FaRegTrashAlt } from 'react-icons/fa'
+import { MdAccountCircle } from 'react-icons/md'
+import { PrimaryButton, AlertButton } from '@/components/button'
+import { Link } from '@/components/link'
+import { Loading } from '@/components/loading'
+import { useGetUserQuery } from '@generated/api'
+
+export default function AccountPage() {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  // TODO: Remove this line after implementing login
+  const tmpUuid = 'a1f4509d-39f6-4de7-921d-201ec3d4578f'
+  const { data, loading, error } = useGetUserQuery({
+    variables: { uuid: tmpUuid }
+  })
+  const user = data?.usersCollection?.edges[0].node
+
+  // TODO: Refactoring error handling using Apollo ErrorLink
+  //       https://www.apollographql.com/docs/react/api/link/apollo-link-error/
+  if (error) {
+    // NOTE: If error is thrown, Next.js will render automatically account/error.tsx
+    throw new Error(error.message)
+  }
+
+  return (
+    <>
+      {loading || !data?.usersCollection ? (
+        <Loading />
+      ) : (
+        <Box
+          as="main"
+          minH={{ base: '86vh', lg: '78vh' }}
+          bg={bg}
+          color={color}
+        >
+          <Container
+            minW={{ base: '100%', lg: 'container.sm' }}
+            display="flex"
+            flexDirection="column"
+            alignItems={{ base: 'space-around', lg: 'flex-start' }}
```

Comment:
The account box looks off-centered. Looks like it's more on the left hand side. I think it's because alignItems is set to flex-start. However, if I apply space-around, the delete button won't be aligned properly, so I need to find a way around it.
<img width="200" alt="Screenshot 2023-12-22 at 12 25 42" src="https://github.com/tabi-memo/frontend/assets/70562492/2ff060fc-4f38-419e-8ec3-0ff4a6ec1b1e">

Reply:
@SachiGoto @Kana-TAGUCHI 
It is difficult to left-align the delete button while keeping the box centered.
So can both be designed to be centered like below?

| My suggestion | Figma |
|----------------|-------|
| <img width="1147" alt="Screenshot 2024-01-03 at 18 59 14" src="https://github.com/tabi-memo/frontend/assets/7115171/824685a1-e70d-4914-ab41-e270f9f414b8"> | ![P-1  Account](https://github.com/tabi-memo/frontend/assets/7115171/8da1d44f-754f-40e0-b77c-92a11e6aacb3) |

Reply:
I just committed and it's all good now!
https://github.com/tabi-memo/frontend/pull/13/commits/e154bc9c1939299350858a7c9d3478e1b35b0502